features/test_feature.py

```diff
@@ -20,7 +20,7 @@
              'thrust-qmc', 'thrust-transform-if', 'thrust-policy', 'thrust-list', 'module-kernel',
              'kernel-launch', 'thrust-gather', 'thrust-scatter', 'thrust-unique_by_key_copy', 'thrust-for-hypre',
              'thrust-rawptr-noneusm', 'driverStreamAndEvent', 'grid_sync', 'deviceProp', 'cub_block_p2',
-             'cub_device', 'activemask', 'complex']
+             'cub_device', 'activemask', 'complex', 'user_defined_rules']
```

Comment:
mark "user_defined_rules" as execution test. 
the replaced API "my_min" and "foo3" are changed to "extern" functions. 
So link error will happen if rules "foo3" and "min" do not work.